third_party/2and3/google/protobuf/json_format.pyi

```diff
@@ -0,0 +1,32 @@
+import sys
+from typing import Any, Dict, Text, TypeVar, Union
+from google.protobuf.message import Message
+
+_MessageVar = TypeVar('_MessageVar', bound=Message)
+
+class Error(Exception): ...
+
+class ParseError(Error): ...
+
+class SerializeToJsonError(Error): ...
+
+def MessageToJson(
+    message: Message,
+    including_default_value_fields: bool = ...,
+    preserving_proto_field_name: bool = ...,
+    indent: int = ...,
+    sort_keys: bool = ...,
+    use_integers_for_enums: bool = ...
+) -> str: ...
+
+def Parse(text: Union[bytes, Text], message: _MessageVar, ignore_unknown_fields: bool = ...) -> _MessageVar: ...
+
+if sys.version_info >= (3, 1):
```

Comment:
What is the reason for the version check? I don't see that in the `.py` file you link to.

Reply:
That was me being misled by https://developers.google.com/protocol-buffers/docs/reference/python/google.protobuf.json_format-module. It doesn't make any mention of `MessageToDict` or `ParseDict`, so I thought they were newer additions, and I saw in the GitHub repo that they're absent in the 3.0.x branch but present in 3.1.x.

However, I did `pip install protobuf` in Python 2.7 just now, and all four methods are there. I removed the version check.